### PR TITLE
Added most user stories and use cases

### PR DESCRIPTION
**Pull request by pixelky**
Missing: Calendar
Other issues: Add test cases, non-functional requirements

Please check your own features. Style is a lot different from draft because we cannot reveal implementation details.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -831,35 +835,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+|`* * *` |new JCRC member |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |JCRC member |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |JCRC member |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |JCRC member |list all persons |
```
Comment (ericyjw) [inline docs/DeveloperGuide.adoc:844]: Not sure if it matters, but the plural for person is people.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -831,35 +835,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+[discrete]
+===Use case: Find person
+
+*MSS*
+
+1. JCRC member requests to find person containing specific name
+2. Hallper shows list of persons with matching name
++
+Use case ends.
+
```
Comment (ericyjw) [inline docs/DeveloperGuide.adoc:952]: If there is no person containing specific name, Hallper shows a message that "There is no one in the list called <name>"??? Or this is not required?

Comment (ericyjw) [reply docs/DeveloperGuide.adoc:952]: Generally is it required to include the "error" message in developer guide?

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -873,10 +1004,207 @@ Use case ends.
...
+** 3a1. Hallper shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Publish calendar
+
+*MSS*
+
+1.
```
Comment (ericyjw) [inline docs/DeveloperGuide.adoc]: To be completed.

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (pixelky) [reply docs/DeveloperGuide.adoc:952]: Nope, too specific

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (pixelky) [reply docs/DeveloperGuide.adoc:844]: I followed the words of the original developer guide, should be fine

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (GilgameshTC) [reply docs/DeveloperGuide.adoc]: Missing Calendar Use Case : 
-------------------------------
[discrete]
=== Use case: publish calendars

*MSS*

1. JCRC member requests to publish calendar
2. Hallper requests for the intended calendar month
3. JCRC member enters the month
4. Hallper saves an output file in the computer

Use case ends.

[discrete]
=== Use case: Add event on Calendar

*MSS*

1. JCRC member requests to add new event to calendar
2. Hallper requests for all details of event (Which month, date, start time, end time, event name)
3. JCRC member enters all details
4. Hallper creates a new event on the calendar

Use case ends.

*Extensions*

[none]
* 3a. Another event clashes with the same timing

Use case ends.

[discrete]
=== Use case: Delete event on Calendar

*MSS*

1. JCRC member requests to delete an event on the calendar
2. Hallper requests for all details of event (Which month, date, event name)
3. JCRC member enters all details
4. Hallper deletes the specified event on the calendar

Use case ends.

*Extensions*

[none]
* 3a. No such event exist

Use case ends.

**Review by GilgameshTC — CHANGES_REQUESTED** [review]
I added onto Eric's review on line 1016.

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (pixelky) [reply docs/DeveloperGuide.adoc]: Code added.

**Review by GilgameshTC — APPROVED** [review]

**Comment by GilgameshTC** [comment]
Looks good! Great Job Kok Yuan!

**Pull request merged**